spec/controllers/advice_controller_spec.rb

```diff
@@ -264,6 +394,7 @@
         expect(result.status).to eq 302
         expect(result).to redirect_to('/advice/edit_advice?id=1')
       end
+    end
```

Comment:
This end should not be here since sets a success flash notice is part of the when the advice is present context.